src/components/nav-menu/styles.scss

```
@@ -0,0 +1,58 @@
+@import 'styles/settings';
+@import 'styles/mixins';
+
+.c-nav-menu {
+  width: 100%;
+  margin: auto;
+  display: flex;
+  justify-content: flex-end;
+  font-family: $main-font;
```

Try to name your settings as the attribute you are using. Then is simpler for everyone to work on the app.
```suggestion
  font-family: $font-family-1;
```